sdks/go/examples/wordcount/wordcount.go

```diff
@@ -146,13 +148,28 @@ func (f *extractFn) ProcessElement(ctx context.Context, line string, emit func(s
 		if len(word) < f.SmallWordLength {
 			smallWords.Inc(ctx, 1)
 		}
-		emit(word)
+		emit(word, 1)
 	}
 }
 
+type formatFn struct {
+	BasicTimer timers.EventTimeTimer
+}
+
 // formatFn is a DoFn that formats a word and its count as a string.
-func formatFn(w string, c int) string {
-	return fmt.Sprintf("%s: %v", w, c)
+func (f *formatFn) ProcessElement(ctx context.Context, t timers.Provider, w string, c int) string {
+	// fmt.Printf("setting timer: %v", f.BasicTimer)
+	f.BasicTimer.SetWithTag(t, "first", time.Now().Add(time.Second*5))
```

Comment:
Since it's an "event time" timer, do we need to set this relative to the "Event time" instead of "now" ?

Reply:
I think you are right. I'll try that